src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3645,12 +3645,10 @@
      * <p>parses as:</p>
      *
      * <pre>
-     * +--TYPE_PARAMETER
-     *     |
-     *     +--IDENT (A)
-     *     +--TYPE_UPPER_BOUNDS
-     *         |
-     *         +--IDENT (Collection)
+     * |--TYPE_PARAMETER -&gt; TYPE_PARAMETER
```

Comment:
Please remove first `|` on each line.

Reply:
Please extend the example.
```java
A extends Collection
```
looks like ` class A extends Collection`.
Please use generic class/method declaration here.

Reply:
Done.